File: objectModel/TypeScript/Persistence/CdmFolder/types/Projections/OperationAddCountAttribute.ts
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.

import { TypeAttribute } from '../TypeAttribute';
import { OperationBase } from './OperationBase';

/**
 * OperationAddCountAttribute class
 */
export abstract class OperationAddCountAttribute extends OperationBase {
    public countAttribute: TypeAttribute;
}

File: objectModel/Python/cdm/persistence/cdmfolder/types/projections/operation_add_count_attribute.py
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License. See License.txt in the project root for license information.

from cdm.persistence.cdmfolder.types.type_attribute import TypeAttribute
from cdm.persistence.cdmfolder.types.projections.operation_base import OperationBase


class OperationAddCountAttribute(OperationBase):
    """OperationAddCountAttribute class"""

    def __init__(self):
        super().__init__()

        self.countAttribute = None  # type: TypeAttribute
